omnibus/Gemfile

```diff
@@ -10,6 +10,8 @@ gem "pedump"
 
 gem "ffi", "< 1.17"
 
+gem "train-core", "< 3.12.5"
```

Comment:
missing omnibus/Gemfile.lock for this pin but also above comment about downgrading